app/javascript/src/components/Notes/Create/index.jsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+
+import { Check } from "@bigbinary/neeto-icons";
+import { Pane, Typography, Button } from "@bigbinary/neetoui/v2";
+import { Input, Textarea, Select } from "@bigbinary/neetoui/v2/formik";
+import { Formik, Form } from "formik";
+import PropTypes from "prop-types";
+
+import formInitialValue from "constants/formInitialValues";
+import formValidationSchema from "constants/formValidationSchemas";
```

Comment:
Please move the constant file in the scope of this directory. 
Place a constant `NOTE_FORM_INITIAL_VALUES` inside `constants.js` inside the `Notes` directory.
Same is the case with `NOTE_VALIDATION_SCHEMA`.